builder/vmware/common/driver_esx5.go

```diff
@@ -567,7 +566,7 @@ func (d *ESX5Driver) VNCAddress(ctx context.Context, _ string, portMin, portMax
 	}
 
 	if vncPort == 0 {
-		err := fmt.Errorf("Unable to find available VNC port between %d and %d",
+		err := fmt.Errorf("unable to find available vnc port between %d and %d",
```

Comment:
VNC here too should stay capitalised I think

Reply:
Resolved in 67054cd.